wifiphisher/common/phishinghttp.py

```diff
@@ -131,7 +133,13 @@ def post(self):
                 logger.info("POST request from %s with %s",
                             self.request.remote_ip, post_data)
             if re.search(constants.REGEX_PWD, post_data, re.IGNORECASE) or \
-            re.search(constants.REGEX_UNAME, post_data, re.IGNORECASE):
+               re.search(constants.REGEX_UNAME, post_data, re.IGNORECASE):
+                if credential_log_path:
+                    with open(credential_log_path, 'a+') as credential_log:
+                        credential_log.write("{} {}".format(
+                            datetime.datetime.now(),
```

Comment:
You can put a better format here. Instead of `datetime`, something like: `time.strftime("%Y-%m-%d %H:%M:%S")`